src/sgame/sg_main.cpp

```diff
@@ -1556,15 +1555,9 @@ static void GetAverageCredits( int teamCredits[], int teamValue[] )
 		teamValue[ team ] = 0;
 	}
 
-	for ( playerNum = 0; playerNum < MAX_CLIENTS; playerNum++ )
+	for ( playerNum = 0; playerNum < level.maxclients; playerNum++ )
 	{
-		playerEnt = &g_entities[ playerNum ];
-		client = playerEnt->client;
-
-		if ( !client )
```

Comment:
Needs the check for whether the client exists

Reply:
The client always exists as as long as clientNum < level.maxclients. The initalisation code does that, and even `ClientDisconnect` sets the value again after a memset: `ent->client = level.clients + clientNum;`

Reply:
So it's a pre-existing bug that the test didn't work. It's probably harmless to the calculation's correctness, but better not to read out data from nonexistent clients.

Reply:
Seems like it wasn't harmless to the calculation correctness (avegare=sum/number), but the value was also never read. Added a commit for that.